keps/sig-node/3066-subsecond-probes/README.md

```diff
@@ -0,0 +1,859 @@
+  - Condition name:
+  - Other field:
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+
+TBD
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+### Dependencies
+
+This section will be completed when targeting beta graduation to a release.
+
+###### Does this feature depend on any specific services running in the cluster?
+
+<!--
+Think about both cluster-level services (e.g. metrics-server) as well
+as node-level agents (e.g. specific version of CRI). Focus on external or
+optional services that are needed. For example, if this feature depends on
+a cloud provider API, or upon an external software-defined storage or network
+control plane.
+
+For each of these, fill in the following—thinking about running existing user workloads
+and creating new ones, as well as about cluster-level services (e.g. DNS):
+  - [Dependency name]
+    - Usage description:
+      - Impact of its outage on the feature:
+      - Impact of its degraded performance or high-error rates on the feature:
+-->
+### Scalability
+
+###### Will enabling / using this feature result in any new API calls?
+
+Enabling / using this feature will not result in any new API calls.
+See earlier KEP text regarding limits and scaling (to zero the millisecond offsets)
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No new scalability API types.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No new calls to the cloud provider will be provided.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+Enabling this feature will not necessarily result in increasing size or count of
+the existing API objects. (3 optional omit if nil *int32 fields for each probe)
```

Comment:
I'd explicitly call out what's the potential impact (size) of having a `*int32` set per container.

Reply:
ok.. will update 
```
package main

import (
	"fmt"
	"reflect"
	"runtime"
)

func main() {
var num int32 = 0
var ptr *int32 = &num

i := reflect.TypeOf(num)
fmt.Printf("size of int32 %d bytes\n", i.Size())

p := reflect.TypeOf(ptr)
fmt.Printf("size of a ptr on this platform %d bytes\n", p.Size())

fmt.Printf("The cost of 3 optional omit if nil *int32 fields is %d bytes for each probe on %s %s\n", (p.Size()+i.Size())*3, runtime.GOOS, runtime.GOARCH)

fmt.Println("The potential size increase of the serialized json struct is dependent on the number of chars it takes to represent the new values if used.") 

}
```
size of int32 4 bytes
size of a ptr on this platform 8 bytes
The cost of 3 optional omit if nil *int32 fields is 36 bytes for each probe on linux amd64
The potential size increase of the serialized json struct is dependent on the number of chars it takes to represent the new values if used.

Reply:
To the doc please.